checker_prv.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CHECKER_PRV_H_
+#define CHECKER_PRV_H_
+
+#include <stdio.h>
+#include <assert.h>
+/* Macros */
+#define TEMP_MIN 0
+#define TEMP_GENRAL_MAX 45
+#define TEMP_HOME_MAX 80
+#define TEMP_PUBLIC_MAX 210
+#define TEMP_EV_MAX 150
+#define MAXNUMBEROFSECTORS 4
+#define SOC_MIN (float)20
+#define SOC_MAX (float)80
+#define RATEOFCHARGE (float)0.8
+#define MIN 0
+#define MAX 1
+#define MAX_WARNING_TEXT_CHARS 40
+#define CALCTOLERANCE(x) (float)(x*(0.05))
+#define CEL 0 /* celcius */
+#define FAHR 1 /* Fahrenheit */
+#define CELTOFAHR(Y) (float)((Y*(9/5))+32)
+extern unsigned int accumateReportCnt;
```

Comment:
What does it take to avoid global's?
Global variables make it harder to track the side effects of code.
Like resetting, etc.